en/flight_controller/pixhawk6c_mini.md

```diff
@@ -0,0 +1,206 @@
+- Newly designed integrated vibration isolation system to filter out high frequency vibration and reduce noise to ensure accurate readings
+- IMUs are temperature-controlled by onboard heating resistors, allowing optimum working temperature of IMUs&#x20;
+
+## Technical Specification
+
+### **Processors & Sensors**
+
+- FMU Processor: STM32H743&#x20;
+  - 32 Bit Arm® Cortex®-M7, 480MHz, 2MB memory, 1MB SRAM&#x20;
+- IO Processor: STM32F103
+  - &#x20;32 Bit Arm® Cortex®-M3, 72MHz, 64KB SRAM&#x20;
+- On-board sensors&#x20;
+  - &#x20;Accel/Gyro: ICM-42688-P&#x20;
+  - Accel/Gyro: BMI055&#x20;
+  - Mag: IST8310&#x20;
+  - Barometer: MS5611
+
+### **Electrical data**
+
+- Voltage Ratings:
+  - Max input voltage: 6V
+  - USB Power Input: 4.75\~5.25V
+  - Servo Rail Input: 0\~36V
+- Current Ratings:
+  - `TELEM1`` Max output current limiter: 1A
+  - All other port combined output current limiter: 1A
+
+### **Mechanical data**
+
+- Dimensions: 53.3 x 39 x 16.2 mm
+- Weight : 39.2g
+
+### **Interfaces**
+
+- 16- PWM servo outputs (8 from IO, 6 from FMU)
+- 3 general purpose serial ports
+  - `TELEM1` - Full flow control, separate 1A current limit
+  - `TELEM2` - Full flow control
+- 2 GPS ports
+  - GPS1 - Full GPS port (GPS plus safety switch)
+  - GPS2 - Basic GPS port
+- 1 I2C port
+  - Supports dedicated I2C calibration EEPROM located on sensor module
+- 2 CAN Buses
+  - CAN Bus has individual silent controls or ESC RX-MUX control
+- 1 Debug ports:
+  - FMU Debug Mini
+- Dedicated R/C input for Spektrum/DSM and S.BUS, CPPM, analog / PWM RSSI
+- 1 Power input ports (Analog)
+
+- Other Characteristics:
+  - Operating & storage temperature: -40 ~ 85°c
+
+## Where to Buy
+
+Order from [Holybro](https://holybro.com/products/pixhawk-6c-mini).
+
+## Assembly/Setup
+
+The Pixhawk 4 Mini's port is very similar to the Pixhawk 6C Mini's port.
+Please refer to the [Pixhawk 4 Mini Wiring Quick Start](../assembly/quick_start_pixhawk4_mini.md) as it provides instructions on how to assemble required/important peripherals including GPS, Power Module etc.
+
+## Pinouts
+
+- [Holybro Pixhawk 6C Mini Port Pinout](https://docs.holybro.com/autopilot/pixhawk-6c-mini/pixhawk-6c-mini-ports)
+
+## Serial Port Mapping
+
+| UART   | Device     | QGC Parameter Description | Port Label on FC |
+| ------ | ---------- | ------------------------- | ---------------- |
+| USART1 | /dev/ttyS0 | GPS1                      | GPS1             |
+| USART2 | /dev/ttyS1 | TELEM3                    | N/A              |
+| USART3 | /dev/ttyS2 | N/A                       | FMU Debug        |
+| UART5  | /dev/ttyS3 | TELEM2                    | TELEM2           |
+| USART6 | /dev/ttyS4 | PX4IO                     | I/O PWM Out      |
+| UART7  | /dev/ttyS5 | TELEM1                    | TELEM1           |
+| UART8  | /dev/ttyS6 | GPS2                      | GPS2             |
+
+<!-- See http://docs.px4.io/main/en/hardware/serial_port_mapping.html#serial-port-mapping -->
```

Comment:
FYI, this is just for me - reminds me how we work this out.

I like the addition of the QGC Parameter description column.